OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -175,8 +175,8 @@ public interface INotifyDeployComplete
 
 	public interface INotifyDeployTriggered
 	{
-		void Deploy(Actor self, bool skipMakeAnim);
-		void Undeploy(Actor self, bool skipMakeAnim);
+		void Deploy(Actor self, bool skipMakeAnim, string[] bodyNames);
+		void Undeploy(Actor self, bool skipMakeAnim, string[] bodyNames);
```

Comment:
I vote for removing the boolean, I would think it is expected that if it can't find anything it shouldn't play anything, the same would happen if you defined a DeployType that no WithMakeAnimation trait triggers on though we might want to make a linter for that.